handler/server.go

```diff
@@ -75,7 +75,7 @@ type Server struct {
 	ContentAPILinkHandler     *api.LinkHandler
 	ContentAPIPostHandler     *api.PostHandler
 	ContentAPISheetHandler    *api.SheetHandler
-	ContentAPIOptionHander    *api.OptionHandler
+	ContentAPIOptionHandler    *api.OptionHandler
```

Comment:
@YinTokey Thanks!  There is a mistake here

Reply:
Done.